pkg/apis/pipelinesascode/v1alpha1/types.go

```diff
@@ -59,8 +59,9 @@ type RepositoryRunStatus struct {
 
 // RepositorySpec is the spec of a repo
 type RepositorySpec struct {
-	URL         string       `json:"url"`
-	GitProvider *GitProvider `json:"git_provider,omitempty"`
+	URL              string       `json:"url"`
+	GitProvider      *GitProvider `json:"git_provider,omitempty"`
+	ConcurrencyLimit int          `json:"concurrency_limit,omitempty"`
```

Comment:
Is `0` a valid limit? you may want to use *int32 to see if user has set any value and in the absence, provide a default.

Reply:
i think the code check for 0 for non defined later on but yeah this may be better as pointer and check for nil,